base/setup/reactos/reactos.c

```diff
@@ -1079,14 +1254,48 @@ FileCopyCallback(PVOID Context,
             if (CopyContext->TotalOperations >> 1 == CopyContext->CompletedOperations)
                 DPRINT1("CHECKPOINT:HALF_COPIED\n");
 
-            SendMessageW(CopyContext->hWndProgress, PBM_STEPIT, 0, 0);
+            SendMessageW(UiContext.hWndProgress, PBM_STEPIT, 0, 0);
             break;
         }
     }
 
     return FILEOP_DOIT;
 }
 
+static VOID
+__cdecl
+RegistryStatus(IN REGISTRY_STATUS RegStatus, ...)
+{
+    /* WARNING: Please keep this lookup table in sync with the resources! */
+    static const UINT StringIDs[] =
+    {
+        IDS_REG_DONE,                   /* Success */
+        IDS_REG_REGHIVEUPDATE,          /* RegHiveUpdate */
+        IDS_REG_IMPORTFILE,             /* ImportRegHive */
+        IDS_REG_DISPLAYSETTINGSUPDATE,  /* DisplaySettingsUpdate */
+        IDS_REG_LOCALESETTINGSUPDATE,   /* LocaleSettingsUpdate */
+        IDS_REG_ADDKBLAYOUTS,           /* KeybLayouts */
+        IDS_REG_KEYBOARDSETTINGSUPDATE, /* KeybSettingsUpdate */
+        IDS_REG_CODEPAGEINFOUPDATE,     /* CodePageInfoUpdate */
+    };
+
+    if (RegStatus < _countof(StringIDs))
+    {
+        va_list args;
+        va_start(args, RegStatus);
+        // GetDlgItem(UiContext.hwndDlg, IDC_ITEM)
+        SetWindowResPrintfVW(UiContext.hWndItem, SetupData.hInstance, StringIDs[RegStatus], args);
+        va_end(args);
+    }
+    else
+    {
+        // GetDlgItem(UiContext.hwndDlg, IDC_ITEM)
```

Comment:
```suggestion
```
a preparation for the future? Or forgotten to strip that?

Reply:
forgot to strip, i've removed it.